AdminUI/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# Build stage
+FROM node:lts-alpine as build-stage
+WORKDIR /app
+COPY package*.json /app/
+RUN npm install
+COPY ./ /app/
+ARG configuration=production
+RUN npm run build -- --output-path=./dist/out --configuration $configuration
+
+# Deploy stage
+FROM nginx:1.24
+COPY --from=build-stage /app/dist/out/ /usr/share/nginx/html
+COPY nginx.conf /etc/nginx/conf.d/default.conf
```

Comment:
Why no entrypoint?

Reply:
AS you can see in [this guide](https://www.docker.com/blog/how-to-use-the-official-nginx-docker-image/), the used image provides a Docker _underlying entrypoint_, which removes the need to provide one.

Reply:
Mhh, okay. Seems to be the official way, so go for it. :)